src/org/zaproxy/zap/extension/quickstart/QuickStartPanel.java

```diff
@@ -259,20 +259,36 @@ private String getPlugNHackUrl() {
 	private ZapTextField getConfField () {
 		if (confField == null) {
 			confField = new ZapTextField();
-			confField.setText(getPlugNHackUrl());
 			confField.setEditable(false);
+			updateConfField(Model.getSingleton().getOptionsParam().getApiParam().isEnabled());
 		}
 		return confField;
 	}
 	
+	public void updateConfField(boolean apiState) {
+		if (confField == null) {
+			getConfField();

```

Comment:
This statement can be removed. The field does not need to be updated if it was not initialised nor added to the panel.